### PR TITLE
[webui] Improve UI for login form

### PR DESCRIPTION
**Pull request by srivatsav1998**
Improved the UI for the login form. Previously it appears to be coming out of search form. Now an arrow and toggle feature is added to the login button.
screenshot from 2018-03-14 17-57-35

### CODE REVIEW

File: src/api/app/assets/javascripts/webui/application/bento/script.js [hunk 1]
```diff
@@ -1,42 +1,42 @@
 $(function() {
-  
+
```
Comment (Ana06) [inline src/api/app/assets/javascripts/webui/application/bento/script.js]: Removing this spaces is unrelated, so either remove it or put it in a different commit :wink:

File: src/api/app/assets/stylesheets/webui/application/bento/base.scss [hunk 2]
```diff
@@ -177,6 +177,21 @@ blockquote{border-left:5px solid #690;color:#666;font-style:italic;}
...
     }
 }
+#login-form {
+border-radius: 5px;
+margin-top: 40px;
+opacity: 1;
+}
+#login-form::before {
+content: '';
+display: block;
```
Comment (Ana06) [inline src/api/app/assets/stylesheets/webui/application/bento/base.scss]: why is this needed? :thinking:

File: src/api/app/assets/stylesheets/webui/application/bento/base.scss [hunk 3]
```diff
@@ -177,6 +177,21 @@ blockquote{border-left:5px solid #690;color:#666;font-style:italic;}
         }
     }
 }
+#login-form {
```
Comment (Ana06) [inline src/api/app/assets/stylesheets/webui/application/bento/base.scss]: why not adding this in https://github.com/openSUSE/open-build-service/pull/4661/files#diff-683e130277e4182c9a0b2f8636c08b1aR138 ? :thinking:

**Comment by srivatsav1998** [comment]
Sorry, messed it up a lot. I will create a new pull request with clear commits.

**Comment by marcus-h** [comment]
On 2018-03-15 14:43:07 -0700, srivatsav gunisetty wrote:
> Sorry, messed it up a lot. I will create a new pull request with clear commits.
> 
(jfyi: there is actually no need to open a new PR - just make your
changes and do a forced push (git push --force ...) afterwards (the
advantage is that the discussion is not scattered to the four winds))

**Comment by Ana06** [comment]
@srivatsav1998 you can still reopen it if you want (open a new one is fine this time too). It is much easier to review it to be able to see previous comments. And do not worry about the mess, we all have mess some commits at some point :see_no_evil: :wink: 

If you need help to fix the mess, please ask and we'll help you! :wink:

**Comment by Ana06** [comment]
BTW @srivatsav1998 can you add @sknorr as co-author in your commit? He wrote the `css`, so I think it is fair to share the authorship. You can do this with the new [`creating-a-commit-with-multiple-authors` Github feature](https://help.github.com/articles/creating-a-commit-with-multiple-authors). Your commit message should look something like:

```
[webui] Improve UI for login form

The Search form and Login form aren't distinguishable. Add an arrow to make it visually pleasing.

Closes https://github.com/openSUSE/open-build-service/issues/3118

Co-authored-by: Stefan Knorr <sknorr@suse.de>

**Comment by sknorr** [comment]
@Ana06 Unless you want this for git blame purposes, I really don't mind. :)

**Comment by Ana06** [comment]
@sknorr 

> @Ana06 Unless you want this for git blame purposes, I really don't mind. :)

I think it is fair to recognise authorship even if you don't mind :wink: And the new GitHub feature makes is so easy that there is no excuse any more :joy:
